### ISSUE TITLE
Document the Julian de Wit algorithm

### ISSUE THREAD

**Issue by reubano** [concept-to-clinic/concept-to-clinic#20]
## Overview

Participants in the [Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2017) produced several algorithms that we would like to incorporate. To help facilitate this effort, we also want to add documentation so that contributors can make an educated decision when selecting an algorithm to incorporate.

## Expected Behavior

This documentation should enable people to:
 - view the library dependencies and license
 - understand its pros/cons
 - evaluate its performance/accuracy
 - identify which areas of the codebase to target for improvement

Design doc reference: [Detect and select](https://concept-to-clinic.readthedocs.io/en/latest/design-doc.html#detect-and-select)

## Algorithm info

key | value
----|------
team|Julian de Wit
rank|2
repo|https://github.com/juliandewit/kaggle_ndsb2017
trained models|https://retinopaty.blob.core.windows.net/ndsb3/trained_models.rar
converted branch|
ML engine|Keras
engine-version|
ML backend|Tensorflow
backend-version|
training method|
architecture|
algorithm|
OS|Windows
OS version|
Python version|3.5
CUDA version|
cuDNN version|
notes|

## Technical details

 - This feature should be implemented as a new markdown file in the [`docs folder`](https://github.com/concept-to-clinic/concept-to-clinic/tree/master/docs)

## Acceptance criteria

 - [ ] effective documentation for the above

*NOTE: All PRs must follow the standard PR checklist.*
